tests/test_client.py

```diff
@@ -11,13 +11,13 @@ class TestClient(ClientTestCase):
     def test_version_values(self):
         values = self.client._version_values()
         self.assertEqual('Python', values['language'])
-        self.assertRegexpMatches(values['version'], r'[0-9]+[.][0-9]+[.][0-9]+')
```

Comment:
This was deprecated in the newest Python? :O

Reply:
Correct. More info here: https://docs.python.org/3/library/unittest.html#deprecated-aliases